src/test/test-execute.c

```diff
@@ -623,13 +623,19 @@ static void test_exec_privatenetwork(Manager *m) {
 
 static void test_exec_oomscoreadjust(Manager *m) {
         test(m, "exec-oomscoreadjust-positive.service", 0, CLD_EXITED);
-        test(m, "exec-oomscoreadjust-negative.service", 0, CLD_EXITED);
+        if (detect_container() > 0)
+                test(m, "exec-oomscoreadjust-negative.service", 1, CLD_EXITED);
+        else
+                test(m, "exec-oomscoreadjust-negative.service", 0, CLD_EXITED);
```

Comment:
hmm, so please not that on request of the LXC folks the OOM adjust stuff is nowadays implemented permissively: if we see EPERM or EACCES when adjusting the OOM adjust we will proceed without failure. Hence I don't understand why this change should be necessary?